tool.ts

```diff
@@ -255,6 +287,8 @@ export async function cacheDir(sourceDir: string,
 
     // write .complete
     _completeToolPath(tool, version, arch);
+
+    return destPath;
```

Comment:
i changed this to return the dir, that way caller does not need to subsequently call findLocalTool